plugins/artnet/Makefile.mk

```diff
@@ -41,3 +43,6 @@ plugins_artnet_ArtNetTester_CXXFLAGS = $(COMMON_TESTING_FLAGS)
 plugins_artnet_ArtNetTester_LDADD = $(COMMON_TESTING_LIBS) \
                                     plugins/artnet/libolaartnetnode.la
 endif
+
+EXTRA_DIST += \
```

Comment:
Sorry, I didn't explain very well, you can leave this on one line if it's just one file.